### DISCUSSION TITLE
Welcome to neohtop Discussions!

### DISCUSSION THREAD

**Discussion by Abdenasser** [Abdenasser/neohtop#44] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Codycody31** [comment]
Hello, decided to get at least one comment thrown in here. cool project, awesome to see so many different projects for system monitor's (I primarily go with [btop](https://github.com/aristocratos/btop), [resources](https://github.com/nokyan/resources), and now this). Also, there is a AUR package now ([aur/neohtop](https://aur.archlinux.org/packages/neohtop)).

**Reply by Abdenasser** [reply]
this is amazing, thank you

**Comment by Godfather2006** [comment]
Very good repository from here. Would love to collaborate with you

**Reply by Abdenasser** [reply]
thanks and welcome

**Comment by Burhanverse** [comment]
Hey I was wondering if we can get a improvement on the app icon. I mean honestly the current icon just doesn't really match up with the app. I think if we can get something minimal and catppuccin themed icon that would be way better.

**Reply by Abdenasser** [reply]
I'll try to work on something more attractive and cartoonish, but would also appreciate if you can share suggestions (we can open a new discussion for people to start sharing their suggestions)

**Reply by Burhanverse** [reply]
Well I'm not that good at designing but I'll try to create one or two icons and share with you (you can open a new discussion about the icon submissions or suggestions)

**Reply by Burhanverse** [reply]
hey so umm i tried to create this icon,
![neohtop(3)](https://github.com/user-attachments/assets/2463c2d3-e24b-4c9d-aa2a-29f7d0cabb10)

**Reply by Burhanverse** [reply]
@Abdenasser finally i tried using the icon with a custom icon pack and this looks really cool, leme know your opinion on this 

![Screenshot From 2024-11-10 19-21-12](https://github.com/user-attachments/assets/3f1e9c2d-fe67-49cf-8766-448fc0d4a623)

**Reply by Abdenasser** [reply]
I liked it, but I'd also want to see something funky (maybe a maskot)

**Reply by Abdenasser** [reply]
![octopus](https://github.com/user-attachments/assets/9bacd481-339d-46ee-bd6e-d9396ee18e0c)

something of this kind but a bit minimal

**Reply by Abdenasser** [reply]
@Burhanverse what do you think?

**Reply by Burhanverse** [reply]
Hmm interesting

**Reply by Abdenasser** [reply]
you can play with of these models https://perchance.org/generators
